gschem/scheme/gschem/hook.scm

```diff
@@ -49,3 +49,5 @@
 (define-public action-property-hook %action-property-hook)
 
 (define-public bind-keys-hook %bind-keys-hook)
+
+(define-public switch-action-mode %switch-action-mode)
```

Comment:
I think it is necessary to add documentation to the [`(gschem hook)` module docs](https://github.com/lepton-eda/lepton-eda/blob/master/docs/scheme-api/geda-scheme.texi#L2385).